en/docs/learn/examples/protocol-switching/switching-from-tcp-to-https.md

```diff
@@ -76,9 +89,9 @@ Send the following message via TCP to the TCP listener port.
     </soapenv:Body>
 </soapenv:Envelope>
 ``` 
-In Linux, we can save the above request in a <strong>request.xml</strong> file and use netcat to send the TCP request. 
+In Mac, we can save the above request in a <strong>request.xml</strong> file and use Netcat to send the TCP request. 
```

Comment:
it is there in linux too. why we need to change it.

Reply:
fixed it.